fileio/ft_read_event.m

```diff
@@ -2141,6 +2141,9 @@
     
   case 'plexon_nex'
     event = read_nex_event(filename);
+
+  case 'plexon_nex5'
```

Comment:
This now duplicates the next 3 lines. Please remove

Reply:
Done. Was messed up by auto merge?

Reply:
I generated two new test files with more variations of data formats. The files are:
https://www.neuroexplorer.com/downloads/FieldTripTest1.nex
https://www.neuroexplorer.com/downloads/FieldTripTest1.nex5

Each file has 2 wave variables.

Reply:
My experience is that these types of things are typically user errors :). Thanks for the files, I'll upload them to the fileserver. I guess for now the expected behavior is that they can be read in without issues, right?

Reply:
.nex5 file can be read without issues, but not .nex file. I deliberately made .nex file with some data that would break existing FieldTrip .nex import code. I will work on fixing .nex import in the next pull request.